src/pkg/bundle/deploy.go

```diff
@@ -366,47 +298,48 @@ func (b *Bundle) processOverrideVariables(overrideMap *map[string]map[string]*va
 				setVal := strings.Split(k, ".")
 				if setVal[0] == pkgName && strings.ToUpper(setVal[1]) == v.Name {
 					overrideVal = val
-					v.Source = b.getSourcePath(types.CLI)
+					v.Source = types.CLI
```

Comment:
Instead of having these consts in the `types` package, since they are only used in the `bundle` package, what do you think of defining the `ValueSources` type + the constants in the `bundle` package? IMO `types.CLI` looks odd